oauth2l/__init__.py

```diff
@@ -257,6 +266,21 @@ def _GetCredentialsVia3LO(client_info, credentials_filename=None):
     return credentials
 
 
+def _GetCredentialForServiceAccount(json_keyfile, scopes, 
+                                    credentials_filename=None):
+    with open(json_keyfile, 'r') as json_keyfile_obj:
+        client_credentials = json.load(json_keyfile_obj)
+    credential_store = _GetCredentialStore(credentials_filename,
+                                           client_credentials['private_key_id'],
+                                           ' '.join(sorted(scopes)))
+    credentials = credential_store.get()
+    if credentials is None or credentials.invalid:
+        credentials = service_account.ServiceAccountCredentials \
```

Comment:
silly: the \ is awkward; if nothing else, you could do

    credentials = (
        service_account.ServiceAccountCredentials.form_json_keyfile_dict(

or just make an alias, eg

    ServiceAccountCredentials = service_account.ServiceAccountCredentials
    credentials = ServiceAccountCredentials.from_json_keyfile_dict(...)

(or some other variation)